packages/stegoveritas/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegoveritas
+pkgver=1.1
+pkgrel=1
+pkgdesc='Automatic image steganography analysis tool'
+arch=('any')
+groups=('blackarch' 'blackarch-stego')
+url='https://github.com/bannsec/stegoVeritas/'
+license=('MIT')
+depends=('python' 'python-pillow' 'python-numpy' 'python-python-magic' 'python-prettytable' 
+         'python-exifread' 'python-python-xmp-toolkit' 'python-stegoveritas-binwalk' 'python-pypng'
+         'python-apng' 'perl-image-exiftool' 'p7zip' 'foremost')
+makedepends=('python-setuptools')
+source=("https://github.com/bannsec/stegoVeritas/archive/$pkgver.tar.gz")
+sha512sums=('9b112f803f00bba1ca711dffcae5d8358022a5b38a19dbd46531ba48b8e374e2b6f87d1e5e45fd35e542f7fc6df662f78c21887107b5615ac4afe5d693b25ae6')
+
+build() {
+  cd "$pkgver"
```

Comment:
The result of the extract is a dir with name `stegoVeritas-1.1`, no `1.1`, please fix that in all the `cd`.